docs/create-custom-image-using-existing-docker-image.md

```diff
@@ -40,29 +40,53 @@ mvn clean install
 * Add the following lines to the Dockerfile:
 
 ```bash
-### Apply AMPs to the latest Community repository image.
-FROM alfresco/alfresco-content-repository-community:latest
+# Apply AMPs to the latest Community repository image.
+ARG BASE_IMAGE=alfresco/alfresco-content-repository-community
+ARG VERSION=latest
+
+FROM ${BASE_IMAGE}:${VERSION}
+
+# Need these viariables to give permissions to tomcat (refer Dockerfile below)
+ARG GROUPNAME=Alfresco
+ARG USERNAME=alfresco
+ARG TOMCAT_DIR=/usr/local/tomcat
+
+# Need to switch to root to add the new files
+USER root
 
 ### Copy the AMPs from build context to the appropriate location for your application server
-COPY amps /usr/local/tomcat/amps
+COPY amps ${TOMCAT_DIR}/amps
 
 ### Install AMPs on alfresco
-RUN java -jar /usr/local/tomcat/alfresco-mmt/alfresco-mmt*.jar install \
-              /usr/local/tomcat/amps/alfresco-bulk-import-*.amp /usr/local/tomcat/webapps/alfresco -nobackup -force
+RUN java -jar ${TOMCAT_DIR}/alfresco-mmt/alfresco-mmt*.jar install \
+              ${TOMCAT_DIR}/amps ${TOMCAT_DIR}/webapps/alfresco -directory -nobackup -force
+
+# Need to have all Tomcat files owned by root with group GROUPNAME as mentionned (refer Dockerfile below)
```

Comment:
Please correct to:
"All files in the tomcat folder must be owned by root user and Alfresco group as mentioned in the parent Dockerfile"